src/Randomizer.SMZ3.Tracking/AutoTracking/GameInteractor.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Randomizer.Shared;
+using Randomizer.SMZ3.Tracking.Configuration;
+using Randomizer.SMZ3.Tracking.VoiceCommands;
+
+namespace Randomizer.SMZ3.Tracking.AutoTracking
+{
+    public class GameInteractor
+    {
+        private readonly AutoTracker _autoTracker;
+        private readonly Tracker _tracker;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="GameInteractor"/>
+        /// class.
+        /// </summary>
+        /// <param name="tracker">The tracker instance.</param>
+        /// <param name="autoTracker">The auto tracker to associate with this module</param>
+        public GameInteractor(Tracker tracker, AutoTracker autoTracker)
+        {
+            _tracker = tracker;
+            _autoTracker = autoTracker;
+            tracker.GameInteractor = this;
+            autoTracker.GameInteractor = this;
```

Comment:
This feels wrong. Is there no other way to do this?

Reply:
Yeah, I agonized over this a bit last night on the best way to do this. Unfortunately because Tracker itself is spawned by the TrackerFactory and AutoTracker is scoped, the options were a bit limited and I went back and forth on how exactly to do this.

However, I have cleaned it up a bit by changing it to be a TrackerModule so that it can be automatically created and changed it so that it accesses the AutoTracker via the property on Tracker. Maybe not the most ideal, but it's not quite as gross.

Reply:
I can probably fix this in my own refactor branch, as I made Tracker once again injectable.